gulp/build.js

```diff
@@ -96,8 +101,28 @@ module.exports = {
 
 		const writeFilesPromise = Promise.all(_.map(allExports, (exportName) => {
 			return new Promise((resolve, reject) => {
-				const exportCode = `export * from '${importMap[exportName]}';`;
-				// for some reason this doesn't pickup our .babelrc even though it's supposed to
+				const specifierPath = importMap[exportName].path;
+				const specifierType = importMap[exportName].type;
+
+				// Sorry, this is a little hard to read, it's effectively a switch
+				// statement on specifierType
+				const exportCode = specifierType === 'ImportSpecifier' ? `
```

Comment:
i think it would be cleaner with our usual
```
condition
? then
: else
```
format, esp. if combined with https://github.com/declandewet/common-tags#stripindent